Directory.Build.props

```diff
@@ -28,6 +28,11 @@
 
     <DashboardPublishedArtifactsOutputDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'DashboardArtifacts', '$(Configuration)'))</DashboardPublishedArtifactsOutputDir>
   </PropertyGroup>
+  
+ <!-- Source Build properties -->
```

Comment:
This isn't a "source build property". This is going to put stuff in our NuGet packages. Can you move it up to be together with `PackageIconFullPath` and `PackageLicenseExpression`?